op_tests/op_benchmarks/triton/bench_gemm_a8w8_per_token_scale.py

```diff
@@ -0,0 +1,167 @@
+        raise NotImplementedError(f"{args.metric} is not supported")
+
+    line_names = ["Triton"]
+    line_vals = ["triton"]
+    benchmark = triton.testing.Benchmark(
+        x_names=x_names,
+        x_vals=x_vals_list,
+        line_arg="provider",
+        line_vals=line_vals,
+        line_names=line_names,
+        styles=[("green", "-")],
+        ylabel=ylabel,
+        plot_name="GEMM A8W8 Benchmark",
+        args={"metric": args.metric},
+    )
+
+    @triton.testing.perf_report([benchmark])
+    def bench_gemm_a8w8_per_tensor_scale(M, N, K, metric, provider):
+        c_dtype = torch.bfloat16
+        x, weight, x_scale, w_scale, y = generate_gemm_a8w8_per_token_scale_inputs(
+            M, N, K, output=True
+        )
+        # flops
+        flops = 2.0 * M * N * K
+        # memory transfer
+        mem_read = (M * K) * x.element_size() + (N * K) * weight.element_size()
+        mem_write = (M * N) * 2  # TODO: Fix for c_dtype != bf16
+        mem = mem_read + mem_write
+
+        ms = triton.testing.do_bench(
+            lambda: gemm_a8w8_per_token_scale(
+                x, weight, x_scale, w_scale, c_dtype, y
+            ),  # noqa: E731
+            warmup=25,
+            rep=100,
+        )
+
+        # Return exactly one scalar depending on which metric is active
+        if metric == "time":
+            return ms
+        elif metric == "throughput":
+            tflops = flops / ms * 1e-9
+            return tflops
+        elif metric == "bandwidth":
+            bandwidth = mem / (ms * 1e-3) * 1e-9  # GB/s
+            return bandwidth
+        else:
+            raise ValueError("Unknown metric: " + metric)
+
+    bench_gemm_a8w8_per_tensor_scale.run(save_path=".", print_data=True)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        prog="Benchmark A8W8 GEMM",
+        allow_abbrev=False,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    available_models = get_available_models()  # Dynamically load model names
+    model_help = (
+        "Model name to benchmark. Select from: ["
+        + ", ".join(available_models)
+        + "]. Use 'all' to benchmark all models or leave blank for the default benchmark script."
+    )
+    parser.add_argument(
+        "--model-configs",
+        type=str,
+        default="utils/model_configs.json",
+        help="Model config json file.",
+    )
+    parser.add_argument("--model", type=str, help=model_help)
+    parser.add_argument(
+        "-M",
+        type=int,
+        default=4096,
+        help="M dim of model benchmark if only one model is under test",
+    )
+    parser.add_argument(
+        "--shape",
+        type=int,
+        nargs=3,
+        metavar=("M", "N", "K"),
+        help="user-defined shape to benchmark",
+    )
+    parser.add_argument(
+        "--metric",
+        type=str,
+        choices=["time", "throughput", "bandwidth"],
+        default="throughput",
+        help="metric to plot",
+    )
```

Comment:
For gemm benchmarks we have a common parser that William added. Please use that as it has some required cmd line args. Also, we need shape and model benchmarks. See other gemm benchmarks. This is a requirement because these scripts are now used in performance CI